src/common/routes.js

```diff
@@ -261,16 +267,36 @@ const routes = [
             exact: true,
             component: PaymentCampaign,
           },
+          {
+            path: '/rewards/payables/@:userName/:reservationPermlink',
```

Comment:
we can union '/rewards/payables/@:userName' and '/rewards/payables/@:userName/:reservationPermlink' routes